lib/classes/ssl/class.lescript.php

```diff
@@ -494,6 +493,35 @@ protected function log($message)
 	{
 		$this->logger->logAction(CRON_ACTION, LOG_INFO, "letsencrypt " . $message);
 	}
+
+    /**
+     * Ask PHP Curl do fetch data from given URL.
+     *
+     * @param string $url - The URL to grab data from
+     *
+     * @return mixed
+     */
+    protected function getUrlContent($url){
+        $r = [
+            'error' => TRUE,
+            'error_message' => '',
+            'data' => '',
+        ];
+        $ch = curl_init();
+        curl_setopt($ch, CURLOPT_URL, $url);
+        curl_setopt($ch, CURLOPT_USERAGENT, 'Mozilla/4.0 (compatible; MSIE 6.0; Windows NT 5.1; .NET CLR 1.1.4322)');
+        curl_setopt($ch, CURLOPT_RETURNTRANSFER, 1);
+        $data = curl_exec($ch);
+        $httpcode = curl_getinfo($ch, CURLINFO_HTTP_CODE);
+        if($httpcode>=200 && $httpcode<300) {
```

Comment:
You use the acme response later on, no? So you _could_ check it against some spec.